readme.md

```diff
@@ -63,6 +63,7 @@ Returns a pull stream that emits thread objects of public messages.
 * `opts.threadMaxSize`: optional number (default: Infinity). Dictates the maximum amount of messages in each returned thread object. Serves for previewing threads, particularly long ones.
 * `opts.allowlist`: optional array of strings. Dictates which messages **types** to allow as root messages, while forbidding other types.
 * `opts.blocklist`: optional array of strings. Dictates which messages **types** to forbid as root messages, while allowing other types.
+- `opts.following`: optional boolean (default: false). `true` means only threads created by those directly followed will be emitted. Requires `ssb-friends`.
```

Comment:
Perhaps we should add "only threads created by those directly followed (and yourself) will be emitted" just to clarify